app/assets/v2/js/pages/dashboard.js

```diff
@@ -4,6 +4,11 @@ var sidebar_keys = [ 'experience_level', 'project_length', 'bounty_type', 'bount
 
 var localStorage;
 
+document.limit = 20;
+document.draw_distance = 5;
+document.bounty_offset = 0;
+document.finished_appending = false;
```

Comment:
AFAIK setting custom variables to `document` is not a very good idea. It messes up the scope of the variable lookup for `document`. So I think this should be avoided. Something like this can be done I think:

```javascript
var pagination = {
    'limit': 20,
    'draw_distance': 5,
    'bounty_offset': 0,
    'finished_appending': false
};
```
and then use `pagination.limit` instead. Modifying document isn't a good practice.

Reply:
Yea, I agree, but there were already a bunch of variables set on document, so I just followed what was already done so if you guys want to change it at least its consistent